src/test/java/uk/gov/homeoffice/digital/sas/timecard/testutils/CommonUtils.java

```diff
@@ -1,11 +1,28 @@
 package uk.gov.homeoffice.digital.sas.timecard.testutils;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import java.time.LocalDateTime;
 import java.time.ZoneOffset;
 import java.util.Date;
+import uk.gov.homeoffice.digital.sas.timecard.model.TimeEntry;
 
 public final class CommonUtils {
   public static Date getAsDate(LocalDateTime dateTime) {
     return Date.from(dateTime.toInstant(ZoneOffset.UTC));
   }
+
+  public static String generateMessageKey(TimeEntry timeEntry) {
+    return timeEntry.getTenantId()  + ":" + timeEntry.getOwnerId();
+  }
+
+  public static LocalDateTime createLocalDateTime() {
+    return LocalDateTime.of(2022, 1, 1, 9, 0, 0);
+  }
+
+  public static  String timeEntryAsJsonString(final Object obj) throws JsonProcessingException {
```

Comment:
Slight inconsistency between method name and parameter type. 
Name suggests the method is strictly for converting a timeEntry object to json string.
Parameter suggests any object can be converted to a json string.

Reply:
Update method name to

`objectAsJsonString`